pkg/cmd/cli/cmd/set/set.go

```diff
@@ -57,3 +59,29 @@ func NewCmdSet(fullName string, f *clientcmd.Factory, in io.Reader, out, errout
 	templates.ActsAsRootCommand(set, []string{"options"}, groups...)
 	return set
 }
+
+const (
+	setImageLong = `
+Update existing container image(s) of resources.
+`
+
+	setImageExample = `	# Set a deployment configs's nginx container image to 'nginx:1.9.1', and its busybox container image to 'busybox'.
+	%[1]s image dc/nginx busybox=busybox nginx=nginx:1.9.1
+
+	# Update all deployments' and rc's nginx container's image to 'nginx:1.9.1'
+	%[1]s image deployments,rc nginx=nginx:1.9.1 --all
```

Comment:
although I don't know if updating the image for deployment config is a good idea (if it have the ICT)